pom.xml

```diff
@@ -89,20 +89,11 @@
 			</dependency>
 
 			<!-- End OpenMRS core -->
-			
-			<dependency>
-				<groupId>org.mockito</groupId>
-				<artifactId>mockito-all</artifactId>
-				<version>1.8.5</version>
-				<scope>test</scope>
-			</dependency>
-
 			<dependency>
 				<groupId>org.codehaus.groovy</groupId>
 				<artifactId>groovy-all</artifactId>
 				<version>${groovyVersion}</version>
 				<type>jar</type>
-				<scope>provided</scope>
```

Comment:
Did you intentionally remove this scope?

Reply:
I set the scope to provided